### DISCUSSION TITLE
How to change Prism syntax highlighting theme.

### DISCUSSION THREAD

**Discussion by mewhz** [docsifyjs/docsify#1687] [Q&A] [answered]
我使用 docsify 配置 C++ 语法高亮时，应该如何修改语法的主题。
这是我使用 docsify 配置出来的 代码高亮

![uTools_1638976498885](https://user-images.githubusercontent.com/48837161/145233626-1a35f7fa-f6a6-4bda-beb3-866b5cb17313.png)

这是我使用 prism 配置的一些代码主题，请问我应该如何修改成这个样子？

![uTools_1638976793240](https://user-images.githubusercontent.com/48837161/145233961-85bd221a-ddac-4e12-bd11-78d3ea17fde8.png)

**Comment by trusktr** [comment]
This topic belongs in the Discussion section.

**Comment by Koooooo-7** [comment, marked as answer]
Basic Steps: 
- Find a prism theme css lib. such as https://www.npmjs.com/package/prism-themes.
- Import the theme you want.
   ```html
    <link href="https://cdn.jsdelivr.net/npm/prism-themes@1.9.0/themes/prism-vs.min.css" rel="stylesheet" />
  ```

**Reply by mewhz** [reply]
Thank you very much, your answer is of great help to me

**Reply by trusktr** [reply]
Adding to that, generally the convention is that for `prism-xxx.min.css` we can write ` ```xxx `, so for `prism-vs.min.css` we can write ` ```vs `.
